bazel/envoy_internal.bzl

```diff
@@ -51,9 +51,12 @@ def envoy_copts(repository, test = False):
                # debugging info detailing some 1600 test binaries would be wasteful.
                # targets listed in order from generic to increasing specificity.
                # Bazel adds an implicit -DNDEBUG for opt targets.
-               repository + "//bazel:opt_build": [] if test else ["-ggdb3"],
+               #
+               # The GDB currently in the CI build container chokes on dwarf-5, so
+               # force dwarf-4.
+               repository + "//bazel:opt_build": [] if test else ["-ggdb3", "-gdwarf-4"],
                repository + "//bazel:fastbuild_build": [],
-               repository + "//bazel:dbg_build": ["-ggdb3"],
+               repository + "//bazel:dbg_build": ["-ggdb3", "-gdwarf-4"],
```

Comment:
For debug build, it may still produce some compatibility issue as this flag isn't passed to dependencies.

Reply:
Oh, that explains why I still get some errors I think. In practice this change at least let me do normal debugging in envoy code and tests, so it seems like a big improvement over gdb being completely broken.